src/main/java/io/github/externschool/planner/service/ScheduleEventTypeServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package io.github.externschool.planner.service;
+
+import io.github.externschool.planner.entity.schedule.ScheduleEventType;
+import io.github.externschool.planner.repository.schedule.ScheduleEventTypeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+/**
+ * @author Danil Kuznetsov (kuznetsov.danil.v@gmail.com)
+ */
+@Service
+@Transactional
+public class ScheduleEventTypeServiceImpl implements ScheduleEventTypeService {
+
+    private final ScheduleEventTypeRepository eventTypeRepo;
+
+    @Autowired
+    public ScheduleEventTypeServiceImpl(final ScheduleEventTypeRepository eventTypeRepo) {
+        this.eventTypeRepo = eventTypeRepo;
+    }
+
+    @Override
+    @Transactional(readOnly = true)
```

Comment:
this one should be Ok for Spring since it optimizes persistence provider management, but they say (above) that using Spring Data ``For read operations, the transaction configuration readOnly flag is set to true.``

Reply:
I said about this case yesterday

Reply:
it was interesting, thanks